lib/ProductOpener/Ingredients.pm

```diff
@@ -634,26 +681,33 @@ sub extract_ingredients_from_text($) {
 
 			if ($sep =~ /(:|\[|\{|\()/i) {
 
+				# Single separators like commas and dashes
+				my $match = '.*?';	# non greedy match
 				my $ending = $last_separator;
 				if (not defined $ending) {
 					$ending = ",|;|( $dashes )";
```

Comment:
Would it make sense to support `$commas` here? We had other issues where stuff like full-width/chinese commas weren't supported correctly in taxonomy fields

Reply:
Sure, I changed it to $commas